torch_frame/data/dataset.py

```diff
@@ -65,7 +67,10 @@ class DataFrameToTensorFrameConverter:
             column name into stats. Available as :obj:`dataset.col_stats`.
         target_col (str, optional): The column used as target.
             (default: :obj:`None`)
-        sep (str): Separator for multi-categorical columns. (default: :obj:`,`)
+        col_to_sep (Union[str, Dict[str, str]], optional): A dictionary that
+            maps each multi_categorical column to its delimiter or a string
+            indicating the delimiter for all multi_categorical columns. If not
+            specified, then :obj:`,` will be used. (default: :obj:`None`)
```

Comment:
Update the description and disallow `optional`.
If `str`, then we will use that as a separator across columns.

Reply:
Could you update this throughout the code? Thanks.

Reply:
Updated~